src/Symfony/Component/Serializer/Tests/Command/DebugCommandTest.php

```diff
@@ -79,6 +80,41 @@ public function testOutputWithClassArgument()
         );
     }
 
+    public function testOutputWithDiscriminatorMapClass()
+    {
+        $command = new DebugCommand(new ClassMetadataFactory(new AttributeLoader()));
+
+        $tester = new CommandTester($command);
+        $tester->execute(['class' => DummyClass::class], ['decorated' => false]);
```

Comment:
You may can reuse `DummyMessageInterface` instead? If you prefer to create a class, I'd named it with a more meaningful name, such as `DummyWithDiscriminatorMap`

Reply:
using the Interface doesn't work, but i changed the name to be more meaningful.

Reply:
Alright! Just by curiosity, why it didn't work?